tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/CatalogUtil.java

```diff
@@ -792,6 +794,40 @@ public static AlterTableDesc setProperty(String tableName, KeyValueSet params, A
     return alterTableDesc;
   }
 
+  public static AlterTableDesc addPartitionAndDropPartition(String tableName, String[] columns,
```

Comment:
I think that the method name is too long. How about ```addOrDropPartition```? Or do you have any good name?